app/src/main/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreen.kt

```diff
@@ -57,6 +61,12 @@ class BloodSugarHistoryScreen(
 
   private val bloodSugarHistoryAdapter = ItemAdapter(BloodSugarHistoryListItemDiffCallback())
 
+  private val events: Observable<BloodSugarHistoryScreenEvent> by unsafeLazy {
+    addNewBloodSugarClicked()
```

Comment:
Not necessary to do in this PR, but we will also need to add `analyticsName` values to all user generated events in `BloodSugarHistoryScreenEvent` so that they get reported to analytics.